src/content/docs/develop/Tests/WebDriver/ci.md

```diff
@@ -42,55 +49,72 @@ jobs:
       # install system dependencies that Tauri needs to compile on Linux.
       # note the extra dependencies for `tauri-driver` to run which are: `webkit2gtk-driver` and `xvfb`
       - name: Tauri dependencies
+        if: matrix.platform == 'ubuntu-latest'
+        run: |
+          sudo apt-get update &&
+          sudo apt-get install -y
+          webkit2gtk-4.1
```

Comment:
```suggestion
          libwebkit2gtk-4.1-dev
```
I know webkit2gtk-4.1 works with apt-get's glob thingy but it still triggers me x) (also iirc it doesn't work with apt)

Reply:
:| I always use webkit2gtk-4.1 for some reason - crazy copypasta

Reply:
probably because most other distros (arch & fedora for example) don't use the lib prefix, especially arch uses exactly `webkit2gtk-4.1` (if you're still an arch user)